packages/react/src/utils/scriptLoader.ts

```diff
@@ -55,27 +88,46 @@ function getScriptSrc({ frontendApi, scriptUrl, scriptVariant = '' }: LoadScript
 export type ScriptVariant = '' | 'headless';
 
 export interface LoadScriptParams {
-  frontendApi: string;
+  frontendApi?: string;
+  publishableKey?: string;
   scriptUrl?: string;
   scriptVariant?: ScriptVariant;
 }
 
-export function loadScript(params: LoadScriptParams): Promise<HTMLScriptElement | null> {
+export async function loadScript(params: LoadScriptParams): Promise<HTMLScriptElement | null> {
+  // @ts-ignore
+  if (!params.publishableKey && !params.frontendApi && window.__clerkDemoInstance) {
```

Comment:
TODO:

Add `__clerkDemoInstance` and the other "env" variable to the global namespace